website/aws.erb

```diff
@@ -3094,6 +3094,12 @@
                                 <li>
                                     <a href="/docs/providers/aws/d/waf_web_acl.html">aws_waf_web_acl</a>
                                 </li>
+                                <li>
+                                    <a href="/docs/providers/aws/d/waf_ipset.html">aws_waf_ipset</a>
+                                </li>
+                                <li>
+                                    <a href="/docs/providers/aws/d/waf_rule.html">aws_waf_rule</a>
```

Comment:
Sorry, I had snuck in the fix for this (b9c57d16eb8e43ff9664cb8be77bf3fa6c8ba597) right after merging #9479. If you rebase this pull request, this shouldn't be necessary. 👍